sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -15,4 +15,27 @@ export interface GetApplicationDataDto {
    * Application dynamic data.
    */
   data: any;
-}
\ No newline at end of file
+}
+export interface StudentApplicationDTO {
+  applicationNumber: string;
+  studyStartPeriod: string;
+  studyEndPeriod: string;
+  id: number;
+  applicationName: string;
+  award: string;
+  status: string;
+}
+
+export interface ApplicationFormData {
+  selectedLocation: number;
+  selectedProgram?: number;
+  offeringIWillBeAttending?: number;
```

Comment:
I think that we need to rename this variable all over the place.
We are using selectedLocation, selectedProgram and I believe that offeringIWillBeAttending should be selectedOffering.
Thoughts?

Reply:
yes, that will be good, if we are updating this variable, we need to update in form definition  and workflow also

Reply:
I am changing the workflow in my current task. I can update the workflow and the form definition, does it sounds good?
Can you please add the below change to your review also to complete the renaming effort?
![image](https://user-images.githubusercontent.com/61259237/127549110-10a12bb4-d3e2-4a60-9e2a-093b4c266755.png)

Reply:
okay changing to `selectedOffering`